receiver/kafkareceiver/README.md

```diff
@@ -42,6 +42,7 @@ The following settings can be optionally configured:
   - `raw`: (logs only) the payload's bytes are inserted as the body of a log record.
   - `text`: (logs only) the payload are decoded as text and inserted as the body of a log record. By default, it uses UTF-8 to decode. You can use `text_<ENCODING>`, like `text_utf-8`, `text_shift_jis`, etc., to customize this behavior.
   - `json`: (logs only) the payload is decoded as JSON and inserted as the body of a log record.
+  - `azureresourcelogs`: (logs only) the payload is converted from Azure Resource Logs format to OTel format.
```

Comment:
Should this be `azure_resource_logs` instead for consistency in the casing style?